pkg/kab/resource_manager_test.go

```diff
@@ -17,16 +17,17 @@
 package kab_test
 
 import (
-	"github.com/projectriff/cnab-k8s-installer-base/pkg/apis/kab/v1alpha1"
-	"github.com/projectriff/cnab-k8s-installer-base/pkg/kab"
-	"github.com/projectriff/cnab-k8s-installer-base/pkg/kab/vendor_mocks"
-	"github.com/projectriff/cnab-k8s-installer-base/pkg/kubectl/mocks"
 	"errors"
+
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/projectriff/cnab-k8s-installer-base/pkg/apis/kab/v1alpha1"
+	"github.com/projectriff/cnab-k8s-installer-base/pkg/kab"
+	"github.com/projectriff/cnab-k8s-installer-base/pkg/kab/vendor_mocks"
+	mockkubectl "github.com/projectriff/cnab-k8s-installer-base/pkg/kubectl/mocks"
 	"github.com/stretchr/testify/mock"
 	v12 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
`metav1` is a better import name. Along the same lines `v12` should be `corev1`

Reply:
fixed.